lib/ansible/modules/database/aerospike/aerospike_migrations.py

```diff
@@ -0,0 +1,535 @@
+            if len(data_arr) == 1:
+                retval = data_arr[0]
+            else:
+                retval = data_arr
+        return retval
+
+    # creates self._build_list which is a unique list of build versions.
+    def _update_build_list(self):
+        self._build_list = set()
+        for node in self._nodes:
+            build = self._info_cmd_helper('build', node)
+            self._build_list.add(build)
+
+    # just checks to see if the version is 4.3 or greater
+    def _can_use_cluster_stable(self):
+        # if version <4.3 we can't use cluster-stable info cmd
+        # regex hack to check for versions beginning with 0-3 or
+        # beginning with 4.0,4.1,4.2
+        if re.search('^([0-3]|4.[0-2])', min(self._build_list)):
+            return False
+        return True
+
+    # make a unique list of namespaces
+    # TODO: does this work on a rolling namespace add/deletion?
+    # thankfully if it doesnt, we dont need this on builds >=4.3
+    def _update_cluster_namespace_list(self):
+        self._namespaces = set()
+        for node in self._nodes:
+            namespaces = self._info_cmd_helper('namespaces', node)
+            for namespace in namespaces:
+                self._namespaces.add(namespace)
+
+    # create a dict of nodes with their related stats under
+    def _update_cluster_statistics(self):
+        self._cluster_statistics = {}
+        for node in self._nodes:
+            self._cluster_statistics[node] = \
+                self._info_cmd_helper('statistics', node)
+
+    # get a fresh list of all the nodes
+    def _update_nodes_list(self):
+        self._nodes = self._client.get_nodes()
+        if not self._nodes:
+            self.module.fail_json("Failed to retrieve at least 1 node.")
+
+    # returns a bool, does the namespace have migrations for the node passed?
+    # if no node passed, uses the local node or the first one in the list
+    def _namespace_has_migs(self, namespace, node=None):
+        namespace_stats = self._info_cmd_helper("namespace/" + namespace, node)
+        try:
+            namespace_tx = \
+                int(namespace_stats[self.module.params['migrate_tx_key']])
+            namespace_rx = \
+                int(namespace_stats[self.module.params['migrate_tx_key']])
+            test = int(namespace_tx) + 1
+            test = int(namespace_rx) + 1
+        except KeyError:
+            self.module.fail_json(
+                msg="Did not find partition remaining key:" +
+                self.module.params['migrate_tx_key'] +
+                " or key:" +
+                self.module.params['migrate_rx_key'] +
+                " in 'namespace/" +
+                namespace +
+                "' output."
+            )
+        except TypeError:
+            self.module.fail_json(
+                msg="namespace stat returned was not numerical"
+            )
+        if namespace_tx != 0 or namespace_rx != 0:
```

Comment:
Why don't you replace lines 365 to 371 with `return namespace_tx != 0 or namespace_rx != 0`?

Reply:
terse code scares me, but it's not horribly terse so i'll take it